src/envoy/alts/tsi_transport_socket.h

```diff
@@ -35,7 +37,8 @@ class TsiSocket : public Network::TransportSocket,
   /**
    * @param handshaker_factory a function to initiate a TsiHandshaker
    */
-  explicit TsiSocket(HandshakerFactory handshaker_factory);
+  explicit TsiSocket(HandshakerFactory handshaker_factory,
+                     HandshakeValidator handshake_validator);
```

Comment:
I think explicit is not needed anymore.

Reply:
Thanks for the review, also updated the comments.